style.css

```diff
@@ -114,6 +114,12 @@
     padding: 10px;
```

Comment:
- [ ] previous code reviewer suggested making your social media icons similar to the Figma design.
I checked it out and found that this line of code is messing with your icons so if you remove `padding: 10px`, then your icons will be aligned to the `Let's connect` button. then you can give 19 pixels of margin between these icons to make it pixel-perfect to the Figma design. I think that will do the job.
your design

![qwer](https://user-images.githubusercontent.com/96683843/212388476-952b6fab-6ce2-452e-8cc5-354f7abc2f5d.jpg)

figma design

![qw](https://user-images.githubusercontent.com/96683843/212388343-c18d198c-61ca-426c-8425-ef2e5fa34c0c.jpg)